### ISSUE TITLE
Duckypad not working on Windows 11 - RESOLVED - seems to be an AMD issue, try EVERY USB port.

### ISSUE THREAD

**Issue by V22rook** [dekuNukem/duckyPad#115]
I recently purchased a Duckypad and have it working on a computer with Windows 10. However, when I connect it to a computer running Windows 11, it doesn't power on. It will occasionally flash "Please insert SD CARD", even though the card is fully seated. If I plug it back into the Windows 10 computer, it works immediately. Can anyone provide any advice?

**Comment by V22rook** [comment]
Additional Information: While restarting the Windows 11 computer, the duckypad came alive (all profiles, switches glowing, etc), but as soon as Windows 11 booted up, the duckypad went dark again. So it's not an issue with the pad or USB port... it's definitely something with Windows 11.

**Comment by V22rook** [comment]
Post by amck1nnon helped resolve the issue. https://github.com/dekuNukem/duckyPad/issues/66

My computer with Windows 11 is a Corsair One with an AMD chipset, a Ryzen 9 5950X.

I had already tried plugging the duckypad into all of the front USB ports, and the rear upper USB 3.0 and USB-C 3.2 ports. No joy. After reading amck1nnon's post I tried the other USB ports on the back. I don't know why but it worked- the duckypad came to life. The computer didn't seem to be loading any drivers, but I left it a lone for a few minutes. After giving it some time, I unplugged it, and swapped it to the other USB ports, which now all work fine. No problems recognizing the pad. 

Everything seems to be working fine now
